index.js

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line
```

Comment:
- [ ] [Optional] 👍🔧 I encourage you to avoid disabling ESLint in any of your custom files. 🚫 Please consider removing all the disabled ESLint rules. 📝✨ By maintaining active ESLint rules and resolving any issues, you can ensure a cleaner and more standardized codebase. 🧹💻 This will lead to better code quality and readability in your project. 🚀📚 Keep up the good work, and remember to embrace the power of ESLint for code consistency! 🌟😊